skeleton/src/main/java/com/dropbox/kaiken/skeleton/skeleton/core/AppInfoProvider.kt

```diff
@@ -0,0 +1,16 @@
+package com.dropbox.kaiken.skeleton.skeleton.core
+
+import java.util.Locale
+
+data class AppInfoProvider(
+    val appKey: String,
```

Comment:
These probably aren't needed for most apps and seem dbx specific.

Reply:
correct removed